cmd/avrogo/internal/generated_tests/justEnum/schema.avsc

```diff
@@ -0,0 +1,9 @@
+{
+                "name": "MyEnum",
+                "type": "enum",
+                "symbols": [
+                    "a",
+                    "b",
+                    "c"
+                ]
+            }
```

Comment:
Missing newline

Reply:
that code is generated so the new line will be added to all schemas so it's outside of the scope of this PR